flair/trainers/ace_trainer.py

```diff
@@ -120,33 +123,9 @@ def sample_config(self, log_prop_weights: torch.Tensor):
         m = torch.distributions.Bernoulli(one_prob)
         log.info(f"Embedding probabilities: {one_prob}")
         v = m.sample()
-        sample_tries = 1000
-        while v.sum().item() == 0 or self.state_to_key(v) in self.configurations:
-            sample_tries -= 1
-            if sample_tries == -1:
-                break
+        while v.sum().item() == 0 or self.active == v:
```

Comment:
Thanks for the changes! I am now running into a small issue after the first episode is finished. ```RuntimeError: Boolean value of Tensor with more than one value is ambiguous``` because of 'self.active == v' which results in a tensor of booleans. Maybe torch.all(self.active == v) would fix this? Could you double-check this?

Reply:
Hey, sorry I didn't test it properly. I now adjusted it to be closer to the [original](https://github.com/Alibaba-NLP/ACE/blob/main/flair/models/controller.py#L92)
and tested it with a small run.
Now it should run through.